ci/build.yml

```diff
@@ -540,10 +542,8 @@ jobs:
         #
         # release only run on releases and is skipped otherwise.
         if [[ "$(Linux.status)" != "Succeeded"
-            || "$(macOS.status)" != "Succeeded"
```

Comment:
without this, the job fails since the status is skipped.

Reply:
🤦 
Sorry, should have realized that.